app/controllers/bookings_controller.rb

```diff
@@ -1,24 +1,51 @@
 class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show, :edit, :update, :destroy]
+
   def index
+    # booking.all to be swapped with current_user.bookings once user is created
     @bookings = Booking.all
+    # @bookings = current_user.bookings
   end
 
   def show
-    @booking = Booking.find(params[:id])
```

Comment:
Aren't we having a show booking page? With details of the booking?

Reply:
Yep, I create the method **set_booking**  and added **before_action :set_booking, only: [:show, :edit, :update, :destroy]** at the top.  

Is that what you meant?